tests/release/service/happy_path.go

```diff
@@ -42,26 +39,12 @@ var _ = framework.ReleaseServiceSuiteDescribe("Release service happy path", Labe
 
 	var releaseCR *releaseApi.Release
 
-	var testEnvironment = utils.GetEnv("TEST_ENVIRONMENT", releasecommon.UpstreamTestEnvironment)
-
 	BeforeAll(func() {
-		if testEnvironment == releasecommon.DownstreamTestEnvironment {
-			fw, err = framework.NewFramework(utils.GetGeneratedNamespace(devNamespace))
-			Expect(err).NotTo(HaveOccurred())
-			devNamespace = fw.UserNamespace
-			managedNamespace = utils.GetGeneratedNamespace(managedNamespace)
-			kubeAdminClient = fw.AsKubeAdmin
-		} else {
-			var asAdminClient *kubeapi.CustomClient
-			asAdminClient, err = kubeapi.NewAdminKubernetesClient()
-			Expect(err).ShouldNot(HaveOccurred())
-			kubeAdminClient, err = framework.InitControllerHub(asAdminClient)
-			Expect(err).ShouldNot(HaveOccurred())
-			_, err = kubeAdminClient.CommonController.CreateTestNamespace(devNamespace)
-			Expect(err).ShouldNot(HaveOccurred())
-		}
+		fw, err = framework.NewFramework(utils.GetGeneratedNamespace(devNamespace))
+		Expect(err).NotTo(HaveOccurred())
+		devNamespace = fw.UserNamespace
```

Comment:
@jinqi7 @davidmogar FYI I've unified the framework init for both OCP and kind environment. PTAL